auto_nag/scripts/survey_sec_bugs.py

```diff
@@ -14,6 +14,9 @@ def description(self):
         return "Submit survey to assignee of a security bug"
 
     def get_bz_params(self, date):
+        assignee_skiplist = self.get_config("assignee_skiplist", default=[])
```

Comment:
You need to define a skiplist (it'd be `assignee_skiplist`) in `configs/tools.json`:
https://github.com/mozilla/relman-auto-nag/blob/28392fb0b835d9806b764c131b54998aba56d861/auto_nag/scripts/configs/tools.json#L633

But here I guess you want to use:
https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/utils.py#L101
with `negation = True`, in passing the `params` you already created.
This way your params will contain your stuff and something like `(not ( assigned_to=="nobody@mozilla.org" OR assigned_to="..." ... ))`
Normally the function `get_empty_assignees` helps to filter out all the kind of "empty" assignees.